spec/rubocop/cop/rspec/implicit_subject_spec.rb

```diff
@@ -183,4 +183,46 @@ def permits(actions)
       RUBY
     end
   end
+
+  context 'with EnforcedStyle `require_implicit`' do
```

Comment:
For spec completeness, we need a no-offense example of using a named subject (e.g. `expect(instance).to be_good`. Also a block usage example (e.g. `expect { subject }.to change(goodness, :count)`)